webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/Boot.scala

```diff
@@ -536,6 +536,9 @@ class Boot extends Loggable {
     }
   }
 
+  // Run a health check
+  RudderConfig.healthcheckNotificationService.reloadCache()
```

Comment:
I would prefer a `startService` here.

Reply:
a duplicated `healthcheckNotificationService` used only at boot ?

Reply:
no, just the method: `RudderConfig.healthcheckNotificationService.startService`. Or `start` or `init`